Imagekit/Imagekit.cs

```diff
@@ -328,5 +329,44 @@ public int PHashDistance(string firstHex, string secondHex)
         {
             return Calculation.GetHammingDistance(firstHex, secondHex);
         }
+
+        /// <summary>
+        /// Generate Auth params for client-side upload 
+        /// </summary>
+        /// <param name="token">Random Token String)</param>
+        /// <param name="expire">Expire time for the token</param>
+        /// <returns>Returns Authparams including token, expiry time & signature.</returns>
+        public AuthParamResponse GetAuthenticationParameters(string token = null, string expire = null)
+        {
+            var DEFAULT_TIME_DIFF = 60 * 30;
+
+            AuthParamResponse authParameters = new AuthParamResponse();
+            authParameters.token = token;
+            authParameters.expire = expire;
+            authParameters.signature = "";
+
+            if (string.IsNullOrEmpty(this.privateKey))
+            {
+                return authParameters;
+            }
```

Comment:
Please address missing test case warning from Codecov for newly added source code.
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/45416977/210040552-1cc9f650-2cca-4836-a966-6ac2dabbc3a2.png">
<img width="821" alt="image" src="https://user-images.githubusercontent.com/45416977/210040562-cb5beabb-49d5-4612-9501-c19891735f7a.png">